test/test_images/probe_helper/probe_helper.go

```diff
@@ -63,10 +88,40 @@ func forwardFromProbe(ctx context.Context, c cloudevents.Client, receivedEvents
 	}
 }
 
+type cloudEvent struct {
+	ID string `json:"ce-id"`
+}
+
+type message struct {
+	Attributes cloudEvent `json:"attributes"`
+}
+
+type eventData struct {
+	Message message `json:"message"`
+}
+
```

Comment:
are there similar types in the pubsub package?

Reply:
Yes, for the Pub/Sub Message. I couldn't find out a clean way to read the inner CloudEvent from the Pub/Sub Attributes map through, so I just access the "ce-id" key now.